aayushi/src/components/BackGround.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+
+export const BackGround = ({ type }) => {
+  return (
+    <div className="">
+      <div className="w-full h-screen">
```

Comment:
Check here if the two divs are not just one after another without any specific use else remove the unwanted div.